src/main/java/edu/mit/dos/object/ObjectService.java

```diff
@@ -86,6 +86,22 @@ public String create(@RequestParam("handle") String handle,
         return String.valueOf(p.getOid());
     }
 
+    @RequestMapping(value = "/object", method = RequestMethod.DELETE)
+    public void deleteObject(@RequestParam("oid") String oid) {
+        final DigitalObject retrievedDigitalObject = objectJpaRepository.findByOid(Long.valueOf(oid));
+        if (retrievedDigitalObject == null){
+            logger.debug("Error - digital object not found:{}", oid);
+        }
+        final String mainBucket = "https://dos-stage.s3.amazonaws.com/";
```

Comment:
This is a sloppy `dos-stage`-specific solution to the fact that deleteObject needs `{UUID}/{FILE_NUM}` rather than what is returned by getPath(), `{BASE_URL}/{UUID}/{FILE_NUM}`. What is the best way to dynamically get this base URL from S3?

Reply:
Added code related to a baseurl property after a slack chat with @osmandin. Everything is still functioning